app/assets/javascripts/discourse/app/routes/groups-index.js

```diff
@@ -14,11 +14,21 @@ export default class GroupsIndexRoute extends DiscourseRoute {
     return I18n.t("groups.index.title");
   }
 
-  model(params) {
-    return params;
+  async model(params) {
+    const controller = this.controllerFor(this.routeName);
+    controller.set("isLoading", true);
```

Comment:
Now that we're loading data properly, I don't think `isLoading` in the controller should be required. Instead, all our standard loading tooling will be used (e.g. slider/spinner depending on the site setting).

If we remove the `isLoading` logic here, and remove the ConditionalLoadingSpinner in the template, does everything still seem good?